src/main/game/Board.java

```diff
@@ -83,53 +72,91 @@ public boolean findWin(String playerMark) {
         return false;
     }
 
+    public Line diagonalOne() {
+        Line line = null;
+        for (int i = 0; i < size; i += 2) {
+            if (size == 3) {
```

Comment:
Checking the size gets you through today, but what about 5x5? 6x6? It adds a lot of code.
There has to be some mathy trick! For example:
If you number each row and column individually e.g. for 3x3, you'd get rows 0, 1, 2 and columns 0, 1, 2.
One of your diagonals is ALWAYS on same pairs: (0,0), (1,1), (2,2)...

Maybe that helps?